src/pages/app/sidebar/projects/ProjectList.svelte

```diff
@@ -36,9 +36,6 @@
     if (!equal($pinStore, pinned)) $pinStore = sortPins(pinned);
```

Comment:
This line was supposed to prevent the update loop, but it clearly wasn't.

Reply:
Equality is as hard to check as it is to achieve.

Reply:
I think the computed value was the issue, though. It told svelte to set the store value every time the store value changed, so it was always behind.